README.md

```diff
@@ -33,6 +33,7 @@ service/environment.
 - `NotToBeDisplayedDomainList`(optional) - It will be used when logging host name by excluding these strings in service name attribute
 	e.g. domain: ".xyz.com", host: "abc.xyz.com" will be logged as "abc" only    
 - `ExcludedPathList`(optional) - Path list that is not needed for tracing. Each item must start with "/".
+- `Create128BitTraceId` - Create new traces using 128 bit (32 hex character) traceId
```

Comment:
@jcarres-mdsol @adriancole Added bool config Create128BitTraceId to control client if we need to create 128bit or 64 bit traceid

FYI @kenyamat @hkurniawan-mdsol